react/Autosuggest/Autosuggest.js

```diff
@@ -122,10 +124,10 @@ export default class Autosuggest extends Component {
     };
 
     const allInputProps = {
```

Comment:
I actually thought there was a bug here for a while because this had been incorrectly named in the past. This shouldn't be `allInputProps` as it's not for the input—it should be `textfieldProps`.